sous-chef/tests/e2e_tests/test_e2e.py

```diff
@@ -94,8 +94,8 @@ def test_e2e(
             return_value=frozen_due_datetime_formatter.anchor_datetime,
         ):
             with patch.object(RecipeBook, "__post_init__", return_value=None):
-
-                assert_equal_dataframe(self._run_menu(), get_final_menu())
+                final_menu = self._run_menu()
```

Comment:
- [ ] Fix e2e tests -> not fully working atm

Reply:
@AkagiShigeru can you confirm that these now succeed locally for you?
Should we invest time to make sure these run with the CI, so we don't run into problems again?
Tbf, we haven't had as many issues in the past year with upgrades, likely due to the other coverage, but they'd still be nice to have to ensure the inputs-outputs work together as expected.

Reply:
Added them to hopefully only on merge to main since they take awhile.